librz/main/rz-find.c

```diff
@@ -15,6 +15,18 @@
 #include <rz_io.h>
 #include <rz_bin.h>
 
+typedef enum {
+	SEARCH_MODE_BYTES = 0,
```

Comment:
I think these should be in `librz/include/rz_search.h` and also documented.

Reply:
No, it won't. this will be used only to select what to do when calling `rz-find [option]`. it's the same for `rz-diff`